words.html

```diff
@@ -901,69 +904,68 @@
 <!-- - - - - - - - - - - - - - - - - -->
 
 <p id="conclusion_0">
-Game theory has shown us the three things we need for the evolution of trust:
+Teoria gier pokazała nam trzy rzeczy, których potrzebujemy do ewolucji zaufania:
 </p>
 <p id="conclusion_1_a">
-1. REPEAT INTERACTIONS
+1. POWTARZANIE INTERAKCJI
 </p>
 <p id="conclusion_1_a2">
-Trust keeps a relationship going,
-but you need the knowledge of possible future repeat interactions <i>before</i> trust can evolve.
+Zaufanie pomaga utrzymać znajomość w dobrej formie,
+ale potrzebna jest też wiedza na temat możliwych przyszłych interakcji <i>zanim</i> pojawi sie zaufanie.
 </p>
 <p id="conclusion_2_a">
-2. POSSIBLE WIN-WINS
+2. MOŻLIWA WYGRANA OBU STRON
 </p>
 <p id="conclusion_2_a2">
-You must be playing a non-zero-sum game,
-a game where it's at least possible that <i>both</i> players can be better off -- a win-win.
+Musisz grać w grę o sumie niezerowej — to gra, w której możliwe jest,<br>
+że <i>obydwaj</i> gracze będą zadowoleni.
 </p>
 <p id="conclusion_3_a">
-3. LOW MISCOMMUNICATION
+3. MAŁO NIEPOROZUMIEŃ
 </p>
 <p id="conclusion_3_a2">
-If the level of miscommunication is <i>too</i> high, trust breaks down.
-But when there's a little bit of miscommunication, it pays to be <i>more</i> forgiving.
+Jeśli nieporozumień jest <i>zbyt</i> wiele, zaufanie zaczyna znikać.
+Jednak gdy nieporozumień jest niewiele, opłaca się być <i>bardziej</i> wyrozumiałym.
 </p>
 <p id="conclusion_4">
-Of course, real-world trust is affected by much more than this.
-There's reputation, shared values, contracts, cultural markers, blah blah blah.
-And let's not forget...
+Oczywiście w prawdziwym świecie na zaufanie ma wpływ znacznie więcej czynników.
+Jest reputacja, wspólne wartości, umowy, markery kulturowe i tak dalej...
+Nie zapominajmy też o...
 </p>
 <p id="conclusion_btn">
-...the <i>biggest</i> lesson. &rarr;
+...<i>najważniejszej</i> lekcji. &rarr;
 </p>
 
 <p id="outro_1">
-If there's one big takeaway<br>
-from <i>all</i> of game theory, it's this:
+Jeśli jest jedna rzecz, którą powinieneś<br>
+wynieść z <i>całej</i> teorii gier, jest to:
```

Comment:
"takeaway" to nie jest, że zapamiętasz, tylko właśnie takie "wyniesienie"

Może tutaj inaczej, bezpośrednio i bardziej swobodniej:
`Jeśli jest jedna rzecz, którą chcielibyśmy, żebyś<br>
wyniósł z tej całej teorii gier, jest to:`